narwhals/dataframe.py

```diff
@@ -568,16 +572,28 @@ def schema(self) -> dict[str, DType]:
 
         Examples:
             >>> import polars as pl
+            >>> import pandas as pd
             >>> import narwhals as nw
-            >>> df_pl = pl.DataFrame(
-            ...     {
+            >>> data = {
             ...         "foo": [1, 2, 3],
             ...         "bar": [6.0, 7.0, 8.0],
             ...         "ham": ["a", "b", "c"],
             ...     }
-            ... )
-            >>> df = nw.DataFrame(df_pl)
-            >>> df.schema  # doctest: +SKIP
+            >>> df_pd = pd.DataFrame(data)
+            >>> df_pl = pl.DataFrame(data)
+
+            We define a library agnostic function:
+
+            >>> def func(df_any):
+            ...     df = nw.from_native(df_any)
+            ...     return df.schema
+
+            You can pass either pandas or Polars to `func`:
+
+            >>> df_pd_schema = func(df_pd)
+            >>> df_pd_schema = func(df_pl)
+
+            Both pandas and Polars returns the same output:
             OrderedDict({'foo': Int64, 'bar': Float64, 'ham': String})
```

Comment:
show the output for each, so it gets tested in CI (see other docstrings for examples)

Reply:
Ok, on it